src/helpers/analytics/events/unload.ts

```diff
@@ -13,6 +14,10 @@ export const track = (
   {pathname}: ReturnType<typeof selector>
 ): AnalyticsEvent | void => {
   return {
+    getEventCapturePayload: () => stateChange({
+      current: 'unload', // TODO - the type only allows string, should it allow null?
```

Comment:
@reedstrm we wanted to represent the unloaded state as a null right? the types coming out of swagger are set to only string values -- is that on your end or a swagger-codegen issue?

Reply:
Probably me, I think to allow it to be null, I'd need to  make it optional. But the semantics are actually that at least one-of-two (current or previous) is required. How about we use a string-nonce to indicate "not existing" - How about "none" since it's not null (which to me means not-known, could be anything - i.e. SQL NULL) but in fact means "there is no previous state" or "there is no current value for this state"

Reply:
ts differentiates between a null or undefined value and having the key not be present in the hash, but in any case using a value of `none` works for me